examples/dft/run_time_dispatching/CMakeLists.txt

```diff
@@ -17,22 +17,22 @@
 # SPDX-License-Identifier: Apache-2.0
 #===============================================================================
 
-# NOTE: user needs to set env var SYCL_DEVICE_FILTER to use runtime example (no need to specify backend when building with CMake)
+# NOTE: user needs to set env var ONEAPI_DEVICE_SELECTOR to use runtime example (no need to specify backend when building with CMake)
 include(WarningsUtils)
 
 
 # Build object from all example sources
 set(DFT_RT_SOURCES "")
 # Set up for the right backend for run-time dispatching examples
 # If users build more than one backend (i.e. mklcpu and mklgpu, or mklcpu and CUDA), they may need to
-# overwrite SYCL_DEVICE_FILTER in their environment to run on the desired backend
+# overwrite ONEAPI_DEVICE_SELECTOR in their environment to run on the desired backend
 set(DEVICE_FILTERS "")
 if(ENABLE_MKLGPU_BACKEND OR ENABLE_CUFFT_BACKEND OR ENABLE_ROCFFT_BACKEND OR ENABLE_PORTFFT_BACKEND)
   list(APPEND DFT_RT_SOURCES "real_fwd_usm")
-  list(APPEND DEVICE_FILTERS "gpu")
+  list(APPEND DEVICE_FILTERS "level_zero:gpu")
```

Comment:
I am worried about this. Could you please check CUFFT and ROCFFT backends?

Reply:
Done in [1c6b27b](https://github.com/oneapi-src/oneMKL/pull/484/commits/1c6b27b68705b02f0f04a802242c3f6b10f87ceb). Testing was performed on a temporary branch and the only failures seen were already present in the base interfaces job.